server/controllers/reviewDataConroller/reviewDataRepository.js

```diff
@@ -0,0 +1,30 @@
+const { ReviewData } = require('../../db/models/Review');
+
+class ReviewDataRepository {
+    constructor(workspaceId) {
+        this.workspaceId = workspaceId != null ? workspaceId : null;
```

Comment:
Yes, as a habit I usually just use `!= null` to consider both `null` and `undefined` rather than using `!==` to choose one or the other. But since this caught your attention, I'm thinking that it would be okay for me to be more specific here. I will have another look through my updated API code. Thanks Josh.